From b8afd0250a310606460872ca5b5532001463828d Mon Sep 17 00:00:00 2001
From: Alon Levy <alevy@redhat.com>
Date: Mon, 3 May 2010 12:56:00 -0300
Subject: [PATCH 06/15] spice-vmc: all variables of type SpiceVMChannel renamed to svc

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1272891368-32706-6-git-send-email-alevy@redhat.com>
Patchwork-id: 8965
O-Subject: [RHEL-6 PATCH 05/13] spice-vmc: all variables of type SpiceVMChannel
	renamed to svc
Bugzilla: 576488
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

From: alon.levy <alevy@redhat.com>

BZ: 576488
---
 hw/spice-vmc.c |   36 ++++++++++++++++++------------------
 1 files changed, 18 insertions(+), 18 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/spice-vmc.c |   36 ++++++++++++++++++------------------
 1 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/hw/spice-vmc.c b/hw/spice-vmc.c
index a225953..2b741a0 100644
--- a/hw/spice-vmc.c
+++ b/hw/spice-vmc.c
@@ -65,32 +65,32 @@ typedef struct SpiceVMChannel {
 static VDObjectRef spice_vmc_interface_plug(
                 VDIPortInterface *port, VDIPortPlug* plug)
 {
-    SpiceVMChannel *d = container_of(port, SpiceVMChannel, interface);
-    if (d->plug) {
+    SpiceVMChannel *svc = container_of(port, SpiceVMChannel, interface);
+    if (svc->plug) {
         return INVALID_VD_OBJECT_REF;
     }
-    d->plug = plug;
+    svc->plug = plug;
     return (VDObjectRef)plug;
 }
 
 static void spice_vmc_interface_unplug(
                 VDIPortInterface *port, VDObjectRef plug)
 {
-    SpiceVMChannel *d = container_of(port, SpiceVMChannel, interface);
-    if (!plug || plug != (VDObjectRef)d->plug) {
+    SpiceVMChannel *svc = container_of(port, SpiceVMChannel, interface);
+    if (!plug || plug != (VDObjectRef)svc->plug) {
         return;
     }
-    d->plug = NULL;
+    svc->plug = NULL;
 
     /* XXX - throw away anything the client has not read */
 
-    if (d->guest_out_ring.bytes != 0) {
+    if (svc->guest_out_ring.bytes != 0) {
         printf("warning: %s: %d unwritten bytes discarded.\n",
-                            __func__, d->guest_out_ring.bytes);
+                            __func__, svc->guest_out_ring.bytes);
     }
-    d->guest_out_ring.read_pos = d->guest_out_ring.write_pos;
+    svc->guest_out_ring.read_pos = svc->guest_out_ring.write_pos;
 
-    if (!d->running) {
+    if (!svc->running) {
         printf("%s: TODO - notify_guest! what to do??\n", __func__);
     }
 }
@@ -137,12 +137,12 @@ static int spice_vmc_interface_read(
     return actual_read;
 }
 
-static void spice_vmc_register_interface(SpiceVMChannel *d)
+static void spice_vmc_register_interface(SpiceVMChannel *svc)
 {
-    VDIPortInterface *interface = &d->interface;
+    VDIPortInterface *interface = &svc->interface;
     static int interface_id = 0;
 
-    if (d->active_interface ) {
+    if (svc->active_interface ) {
         return;
     }
 
@@ -158,17 +158,17 @@ static void spice_vmc_register_interface(SpiceVMChannel *d)
     interface->write = spice_vmc_interface_write;
     interface->read = spice_vmc_interface_read;
 
-    d->active_interface = true;
+    svc->active_interface = true;
     qemu_spice_add_interface(&interface->base);
 }
 
-static void spice_vmc_unregister_interface(SpiceVMChannel *d)
+static void spice_vmc_unregister_interface(SpiceVMChannel *svc)
 {
-    if (!d->active_interface ) {
+    if (!svc->active_interface ) {
         return;
     }
-    d->active_interface = false;
-    qemu_spice_remove_interface(&d->interface.base);
+    svc->active_interface = false;
+    qemu_spice_remove_interface(&svc->interface.base);
 }
 
 
-- 
1.7.0.3

